tests/stdlib/tbitops.nim.cfg

```diff
@@ -1 +1 @@
--d:noUndefinedBitOps
+-d:noUndefinedBitOpts
```

Comment:
What? This should be noUndefinedBitOps somewhere else.

Reply:
I thought it was wrong here, considering that in the `bitops` documentation and in the codes everyone is like` noUndefinedBitOpts`.

Reply:
So the only correct place is in this .cfg.

Because even `noIntrinsicsBitOpts` is Opts.

Reply:
Now, which one would be correct?

- no option of bit intrinsics?
- no intrinsic bitops (bit operations)?

Reply:
My mistake.